frontend/src/components/MediaGallery.tsx/Image.tsx

```diff
@@ -5,26 +5,11 @@ type Props = {
 	mediaAttachment: MediaAttachment
 }
 
-export const focusToObjectFit = (focus: { x: number; y: number }) => {
```

Comment:
This bit actually does something important -- given a user provided focal point, it will zoom the preview image to that location. 

Visually, with this focus logic:
![image](https://user-images.githubusercontent.com/204386/215136433-e8fde60b-23c0-4ea2-afde-8807a7c9ccd4.png)

without:
![image](https://user-images.githubusercontent.com/204386/215136499-3849d238-64a7-4057-849a-3ca40c77c634.png)